README.md

```diff
@@ -1,100 +1,32 @@
-
-In the root directory you can make a virtualenv. It will be excluded from commits by default
-
-```console
-python -m venv venv
-```
-Whenever developing the project you'll want to activate the virtualenv in your terminal. This is platform dependent and there are more comprehensive docs available here: https://docs.python.org/3/library/venv.html
-
-> :warning: If you're using PowerShell on Windows you might need to run `Set-ExecutionPolicy -ExecutionPolicy RemoteSigned -Scope CurrentUser`. More information on execution policy is available in the link above.
-
-| Platform | Shell           | Command to activate virtual environment |
-|----------|-----------------|-----------------------------------------|
-| POSIX    | bash/zsh        | $ source <venv>/bin/activate            |
-|          | fish            | $ source <venv>/bin/activate.fish       |
-|          | csh/tcsh        | $ source <venv>/bin/activate.csh        |
-|          | PowerShell Core | $ <venv>/bin/Activate.ps1               |
-| Windows  | cmd.exe         | C:\> <venv>\Scripts\activate.bat        |
-|          | PowerShell      | PS C:\> <venv>\Scripts\Activate.ps1     |
-
-
-### Setup
-
-Once you have your virtual environment setup and activated you'll want to finish setting up the development environment.
-
-```console
-> pip install --upgrade -r requirements.txt -r requirements-dev.txt -r requirements-win.txt
-> python setup.py develop
-```
-
-This will install any dependencies as well as setting up links on your path to your local source files. Once this is done
-you'll be able to execute the binaries right from your path after any changes to the source without the need to build or
-install anything. If you add new source files you may have to run `python setup.py develop` again to make sure they're linked.
-
-### Running Locally
-
-```
-modlunky2
-```
-
-### Building Distributions
-
-#### PyPI
-```
-python setup.py sdist
-python -m twine upload .\dist\modlunky2-$VERSION.tar.gz
-```
-
-#### EXE
-
-##### PyInstaller
-The default pyinstaller from pip seems prone to false detection from antivirus. Instructions on building your own available at https://pyinstaller.readthedocs.io/en/stable/bootloader-building.html
-
-Alternatively you can `pip install pyinstaller`
-
-See: https://stackoverflow.com/a/52054580
+> :warning: Spelunky 2 doesn't officially support modding. Do not report modding related bugs to Blitworks.
 
-##### Audio DLLs
-We need some ogg/vorbis `dll`'s in order to support extracting files from the FSB soundbank. These can be obtained from https://github.com/HearthSim/python-fsb5/releases
+## Contributing
 
-Drop the `libogg.dll` and `libvorbis.dll` files from python-fsb5_win64.zip into the `dist` directory
+The [documentation for contributing](CONTRIBUTING.md) explains
 
-##### Build
-```
-python build-exe.py
-```
+* How to report bugs
+* Where to discuss changes
+* Setup for development
 
 ## Contributors
```

Comment:
I think it'd be fair to add yourself here though i wonder if there's a better way to have a list like this without it eventually just being most of this readme. There's already a contributor list to the right but some of these people didn't contribute directly to the repo and would still wanna thank them as well.